pkg/repo/mysql/services.go

```diff
@@ -0,0 +1,90 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+)
+
+func (r *Repo) SaveService(ctx context.Context, service *model.Service) error {
+	query := `INSERT INTO service (id, name) VALUES (UUID_TO_BIN(?), ?) ON DUPLICATE KEY UPDATE name = VALUES (name)`
```

Comment:
I'll try to use `sql.Scanner()` and `driver.Value()` in a separate PR to simplify the SQL statements - to avoid using UUID_TO_BIN and BIN_TO_UUID.

Reply:
Actually, google uuid already supports sql.Scanner -https://github.com/google/uuid/blob/master/sql.go#L15. No need to convert BIN_TO_UUID when selecting data.

However, sql.Valuer() returns string represantion which requires UUID_TO_BIN https://github.com/google/uuid/blob/master/sql.go#L57